ui/src/actions/actionTypes.js

```diff
@@ -68,6 +68,10 @@ export const AUTHOR_PUBLICATION_SELECTION_CLEAR =
   'AUTHOR_PUBLICATION_SELECTION_CLEAR';
 export const AUTHOR_SET_ASSIGN_DRAWER_VISIBILITY =
   'AUTHOR_SET_ASSIGN_DRAWER_VISIBILITY';
+export const AUTHOR_PUBLICATION_CLAIMED_SELECTION =
```

Comment:
```suggestion
export const AUTHOR_PUBLICATION_CLAIM_SELECTION =
```

i would say this fits better as naming (applies to all :))